Task70/YandexPages/HomePage.cs

```diff
@@ -0,0 +1,62 @@
+﻿using OpenQA.Selenium;
+using SeleniumExtras.PageObjects;
+using NUnit.Framework;
+using Task70.YandexPages;
+
+namespace Task70
+{
+    class HomePage
+    {
+
+        private IWebDriver driver;
+        private string url = "https://yandex.by/";
+        public HomePage(IWebDriver driver)
+        {
+            this.driver = driver;
+            PageFactory.InitElements(driver, this);
+        }
+
+        [FindsBy(How=How.XPath, Using = "//div[contains(text(), 'Войти')]")]
+        public IWebElement SigninButton { get; set; }
+
+        [FindsBy(How= How.XPath, Using = "//div[contains(@class, 'usermenu-link')]/a[1]/span")]
+        public IWebElement UsernameLink { get; set; }
+
+        [FindsBy(How = How.XPath, Using = "//*[@id='uniq269']/span")]
+        public IWebElement SignoutButton { get; set; }
+
+        public void Open()
+        {
+            driver.Navigate().GoToUrl(url);
+            Assert.AreEqual("Яндекс", driver.Title);
```

Comment:
It will be better to add waiter there to wait until title will have expected value